### PR TITLE
Add support for modal attributes

### LINKED ISSUE #155: Support for modal attributes
It's not currently possible to add custom attributes to modal windows, only to the modal header, body and footer. A `Modal.attr` function would be nice to have.

### PR DESCRIPTION
**Pull request by Munksgaard**
Fixes #155

### CODE REVIEW

File: src/Bootstrap/Modal.elm [hunk 1]
```diff
@@ -282,6 +283,13 @@ hideOnBackdropClick hide (Config ({ options } as conf)) =
     Config { conf | options = { options | hideOnBackdropClick = hide } }
 
 
+{-| Option to trigger close message when the user clicks on the modal backdrop. Default True.
```
Comment (rundis) [inline src/Bootstrap/Modal.elm]: Copy/pasted doc from another function ?

File: src/Bootstrap/Modal.elm [hunk 2]
```diff
@@ -282,6 +283,13 @@ hideOnBackdropClick hide (Config ({ options } as conf)) =
...
+-}
+attr : Html.Attribute msg -> Config msg -> Config msg
```
Comment (rundis) [inline src/Bootstrap/Modal.elm]: Could you change this to attrs (plural) and take a list of html attribute, to keep it in line with other modules.

File: src/Bootstrap/Modal.elm (hunk 1 above)
Comment (Munksgaard) [reply src/Bootstrap/Modal.elm]: Most certainly. I'll fix it :-)

File: src/Bootstrap/Modal.elm (hunk 2 above)
Comment (Munksgaard) [reply src/Bootstrap/Modal.elm]: Sure thing :+1:

**Comment by Munksgaard** [comment]
Do you want it so that you can use multiple applications of `Modal.attrs`, or should each invocation just overwrite the previous one? In other words should `values ++ options.attrs` on line 289 instead by just `values`?

**Comment by girishso** [comment]
+1, I need this!

**Comment by rundis** [comment]
tx! and sorry for the (very) slow response

**Comment by girishso** [comment]
You forgot to publish the new package to elm-packages!

**Comment by rundis** [comment]
Patience is a virtue 😄 I'm fixing a few other things too. A new version will be released when I wrap up the batch of work I made (spare) time for doing work on this. Probably in the not to distant future.

**Comment by girishso** [comment]
Yep, just noticed the other commits, was about to delete my comment!

Thank you for your work on the package! :)

**Pull request merged**